client-js/app/elements/elements.html

```diff
@@ -40,3 +43,4 @@
 <link rel="import" href="labrad-server.html">
 <link rel="import" href="menu-button.html">
 <link rel="import" href="selectable-table.html">
+<link rel="import" href="expandable-input.html">
```

Comment:
Agreed. I had no idea where this convention came from, I has just following suit.